patterns/mod.ts

```diff
@@ -1,3 +1,4 @@
 export * as consensus from "./consensus/mod.ts"
 export * as ink from "./ink/mod.ts"
+export * from "./multisigAccountId.ts"
 export * from "./MultisigRune.ts"
```

Comment:
These should be colocated in one directory.

Reply:
I agree (exposed through `patterns/multisig/mod.ts`), but there are several other changes we should merge first (otherwise, we'll end up rewriting many imports).